docker-compose.yml

```diff
@@ -9,15 +9,7 @@ services:
     ports:
       - "7896:7896"
     environment:
-      ## spring data
       - spring.datasource.url=jdbc:mysql://mysqldb:3306/lpvs
-      - spring.datasource.username=root
-      - spring.datasource.password=lpvsroot
-      ## github data
-      - github.login=
-      - github.token=
-      - github.api.url=https://api.github.com  # for usual GitHub account 
-      - github.secret=LPVS 
```

Comment:
I think these variables are necessary.

Reply:
@o-konoval  Please take a look at Taewan's comment and let me know your opinion.

Reply:
I agree with @tiokim 
We have to consider such case. After adding required parameters in new PR and applying required changes in the source code, we can publish new release v1.0.2 on GitHub.

Reply:
I agree-unfortunately yesterday we did not take in attention such case - with container from registry - so we have to return previous values to docker-compose.yml and update manual. 
Thanks for your notes. 
Then do I start new PR with fix ASAP?

Reply:
@m-rudyk yes, please create new PR.

@m-rudyk  we have to modify the source code to support this scenario (if needed). Please take a look and update the code.